packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -224,6 +233,48 @@ repository.configureClientInstance({
 });
 ```
 
+### Configuring Deadline Secrets Management
+
+By default, [Deadline Secrets Management](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html) is enabled on the `Repository`.
```

Comment:
Same as my other discussion in the upgrade documentation:

>   I know the logic is not yet implemented, but we plan to change this to reflect our plan to have secrets management only enabled when using Deadline version 10.1.19 or higher. What do you think about changing these docs now?

Reply:
Added the version in